pyproject.toml

```diff
@@ -12,12 +12,12 @@ packages = [
 
 [tool.poetry.dependencies]
 python = ">=3.7,<3.10"
-dcicutils = "^6.8.0"
+dcicutils = "^6.9.9.0b2"
 click = "^7.1.2"
 cron-descriptor = "^1.2.31"
 cryptography = "39.0.0" # Required for AWS Cognito JWT decode (PyJWKClient)
 pyDes = "^2.0.1"
-PyJWT = "2.5.0"
+PyJWT = "^2.5.0"
```

Comment:
I would love it if after processing other suggestions here, we also alphabetized these regular and dev dependencies. It's hard to easily visually search for whether something is here or not when the order is random.